src/v2/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -181,6 +160,7 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
       ?.displayPriceRange
 
     if (
+      !isPriceHidden &&
```

Comment:
ditto here, I'm informing them we're going to reduce the number of events because of the bug fix

Reply:
Thank you for adding that! 🙏